### ISSUE TITLE
nice

### ISSUE THREAD

**Issue by NealSavant** [devinCoop/Packt-Full-React-Spring#1]
this is cool, finished user auth between client server?

**Comment by devinCoop** [comment]
Yes but there are some bugs I need to work out with React. I have been doing tutorial after tutorial with React and Spring Security so that I can get better at Jwt, OAuth, Email confirmation, lost password links, etc.
